ospf6d/ospf6_top.c

```diff
@@ -154,8 +213,15 @@ static struct ospf6 *ospf6_create(const char *name)
 	if (vrf) {
 		o->vrf_id = vrf->vrf_id;
 		/* Freed in ospf6_delete */
```

Comment:
This comments needs to be moved down along with o->name = XSTRDUP(MTYPE_OSPF6_TOP, name); It is misleading.

Reply:
Resolved